frontend/src/components/Helper/AsyncCalls.ts

```diff
@@ -29,3 +30,30 @@ export async function querySuggestions(dispatch: Function, query: string) {
 
     dispatch({type: ActionTypes.putSuggestions, suggestions: [/*retrieve suggestions from backend*/]})
 }
+
+/*
+* store selected items in local storage
+*/
+export async function storeItems(items: Item[]) {
+    try{
+        const jsonItems = JSON.stringify(items);
+        await AsyncStorage.setItem('@save_list', jsonItems);
+    } catch (e) {
+        //TODO
```

Comment:
Any idea what kinds of exceptions could pop up?

Reply:
I haven't been able to find a clear list but they seem to be uncommon. [Things like if the device runs out of storage or the OS refuses to fork over more storage for some reason](https://stackoverflow.com/questions/43810759/what-are-the-possible-failure-cases-of-react-natives-asyncstorage).  Any suggestions on an appropriate way to handle potential errors?